### PR TITLE
Added Databases Create Scenario

### PR DESCRIPTION
**Pull request by neelip**
Every file present in `src` is directly moved from `vscode-cosmosdb` except for following files: 
- `constants.ts` --> Removed all unnecessary variables
- `extensionVariables.ts` --> Updated to fit node module format

### CODE REVIEW

File: databases/src/utils/openUrl.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as open from 'open';
+
+export async function openUrl(url: string): Promise<void> {
```
[the remark this answers is not in the document]
Comment (neelip) [reply databases/src/utils/openUrl.ts:8]: We have "openInPortal" exported from vscode-azureextensionui package but not "openUrl". There are few modules/functions which are redundant in the extension repos. I think overall cleanup of these would be a good idea within a different PR.

File: databases/src/create/PostgresAccountWizard/createPostgresServer/PostgresServerSkuStep.ts [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                data: option.sku,
+                group: option.group || localize('addlOptions', 'Additional Options')
+            });
+        });
+        return options;
+    }
+}
+
+const recommendedGroup = localize('recommendGroup', 'Recommended');
+const singleServerSkus: ISkuOption[] = [
```
[the remark this answers is not in the document]
Comment (neelip) [reply databases/src/create/PostgresAccountWizard/createPostgresServer/PostgresServerSkuStep.ts:63]: I have noted this change here: https://github.com/microsoft/vscode-azuretools/issues/1011

File: databases/src/utils/nonNull.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { isNullOrUndefined } from 'util';
+
+/**
+ * Retrieves a property by name from an object and checks that it's not null and not undefined.  It is strongly typed
+ * for the property and will give a compile error if the given name is not a property of the source.
+ */
+export function nonNullProp<TSource, TKey extends keyof TSource>(source: TSource, name: TKey): NonNullable<TSource[TKey]> {
```
[the remark this answers is not in the document]
Comment (neelip) [reply databases/src/utils/nonNull.ts:12]: Created an issue noting this: https://github.com/microsoft/vscode-azuretools/issues/1012

File: appservice/package-lock.json [hunk 4]
```diff
@@ -15197,4 +15197,4 @@
             "dev": true
         }
     }
-}
```
Comment (alexweininger) [inline appservice/package-lock.json:15200]: nit: Add back the space

**Review by alexweininger — APPROVED** [review]

**Pull request merged**